internal/pkg/emit/emit.go

```diff
@@ -39,12 +39,37 @@ func (s *simpleEmitter) Emit(message interface{}) {
 	s.emitch <- message
 }
 
+type multiEmitter struct {
+	emitChs []chan<- interface{}
+}
+
+// Emit emits messages to the configured list of channels.
+func (m *multiEmitter) Emit(message interface{}) {
+	if len(m.emitChs) == 0 {
+		zap.S().Error("emit channels empty")
+	}
+
+	for i := range m.emitChs {
+		if m.emitChs[i] == nil {
+			zap.S().Error("channel misconfigured", "index", i)
+			continue
+		}
+		m.emitChs[i] <- message
```

Comment:
This could block indefinitely if any of the channels block and an exit-point should be implemented (i.e `select{}` block with a timeout).